app/controllers/urls_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'securerandom'
+
+class UrlsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  wrap_parameters false
+
+  attr_reader :urls
+
+  def create
+    uuid = SecureRandom.uuid
```

Comment:
There is no need to generate an id. When you create a job it an id gets assigned to it. That's what we should use.

Reply:
You're right! This makes it so much cleaner